doc/api/util.md

```diff
@@ -2359,6 +2359,24 @@ util.types.isBigInt64Array(new BigInt64Array());   // Returns true
 util.types.isBigInt64Array(new BigUint64Array());  // Returns false
 ```
 
+### `util.types.isBigIntObject(value)`
+
+<!-- YAML
+added: v22.9.0
```

Comment:
```suggestion
added: v10.4.0
```
actually!

Reply:
Thank you for your comment. I just make another commit to correct this.